hikari/impl/bot.py

```diff
@@ -823,6 +824,56 @@ async def wait_for(
     ) -> event_dispatcher.EventT_co:
         return await self._events.wait_for(event_type, timeout=timeout, predicate=predicate)
 
+    async def update_presence(
```

Comment:
Would it not make sense for this method to maybe also be declared on shard aware for the traits?

Reply:
All the traits only include properties, not public methods, which is why I didnt think about adding it. @nekokatt?

Reply:
It should be added.

Reply:
Added :+1: